tests/instrumentation/grpc_tests.py

```diff
@@ -0,0 +1,158 @@
+#  BSD 3-Clause License
+#
+#  Copyright (c) 2024, Elasticsearch BV
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are met:
+#
+#  * Redistributions of source code must retain the above copyright notice, this
+#    list of conditions and the following disclaimer.
+#
+#  * Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the following disclaimer in the documentation
+#    and/or other materials provided with the distribution.
+#
+#  * Neither the name of the copyright holder nor the names of its
+#    contributors may be used to endorse or promote products derived from
+#    this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+#  DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+#  FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+#  DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+#  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+#  CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+#  OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+#  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import pytest  # isort:skip
+
+grpc = pytest.importorskip("grpc")  # isort:skip
+
+import asyncio
+from concurrent import futures
+
+import elasticapm
+from elasticapm.conf import constants
+from elasticapm.conf.constants import TRANSACTION
+from elasticapm.traces import capture_span
+from elasticapm import Client
+from elasticapm.contrib.grpc.client_interceptor import _ClientInterceptor
+from elasticapm.contrib.grpc.server_interceptor import _ServerInterceptor
+from elasticapm.contrib.grpc.async_server_interceptor import _AsyncServerInterceptor
+from tests.fixtures import TempStoreClient, instrument
+from tests.instrumentation.test_pb2 import UnaryUnaryRequest, UnaryUnaryResponse
+from tests.instrumentation.test_pb2_grpc import TestServiceServicer, TestServiceStub, add_TestServiceServicer_to_server
```

Comment:
These modules should be renamed otherwise test runners think these contains tests and runs them